code/controllers/subsystems/ticker.dm

```diff
@@ -383,11 +383,17 @@ var/datum/controller/subsystem/ticker/SSticker
 		LAZYREMOVE(ready_player_jobs[dept], prefs.real_name)
 
 	if(.)
-		total_players_ready--
+		update_ready_count()
+
+/datum/controller/subsystem/ticker/proc/update_ready_count()
+	total_players_ready = 0
+	for(var/mob/abstract/new_player/NP in player_list)
+		if(NP.ready)
+			total_players_ready++
```

Comment:
This works. But it shouldn't be necessary vs. what you had already? Why is incrementing / decrementing when someone actually readies/unreadies inaccurate?